# .yaml config file
#
# The PID gains and controller settings must be saved in a yaml file that gets loaded
# to the param server via the roslaunch file (monoped_control.launch).

monoped:
  # Publish all joint states -----------------------------------
  # Creates the /joint_states topic necessary in ROS
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Effort Controllers ---------------------------------------
  haa_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: haa_joint
    pid: {p: 1.0, i: 0.0, d: 0.5}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  hfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: hfe_joint
    pid: {p: 1.0, i: 0.0, d: 0.5}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  kfe_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: kfe_joint
    pid: {p: 1.0, i: 0.0, d: 0.5}
